### PR TITLE
ProductDetails ContentLoader

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Create a content loader using react-content-loader for the component ProductDetails

#### What problem is this solving?
Keep a concise window size while loading

#### How should this be manually tested?
[workspace](https://contentloader--storecomponents.myvtex.com/samsung-s9/p)

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "product-details",
   "version": "0.1.0",
   "dependencies": {
-    "@vtex/styleguide": "^2.3.0"
+    "@vtex/styleguide": "^2.3.0",
```
Comment (claudivanfilho) [inline react/package.json]: is never used

**Review by claudivanfilho — CHANGES_REQUESTED** [review]

File: react/package.json (hunk 1 above)
Comment (salesfelipe) [reply react/package.json]: 👍

**Review by claudivanfilho — APPROVED** [review]

File: react/ProductDetails.js [hunk 2]
```diff
@@ -89,140 +89,178 @@ class ProductDetails extends Component {
...
-                  </div>
-                )
-              }
+              const shouldDisplayLoader = loading || !product
+              let selectedItem,
+                commertialOffer,
+                sellerId,
+                skuItems,
+                initialItemIndex
+              if (!shouldDisplayLoader) {
```
Comment (lucasecdb) [inline react/ProductDetails.js:129]: wouldn't it be better if you just returned the loading state when `shouldDisplayLoader` is `true`? this way you'd avoid the need for the `let` in the variables declared above.

**Comment by lucasecdb** [comment]
awesome! :clap:

File: react/ProductDetails.js (hunk 2 above)
Comment (salesfelipe) [reply react/ProductDetails.js:129]: This way i have less code and only one point of return

File: react/ProductDetails.js [hunk 3]
```diff
@@ -89,140 +89,178 @@ class ProductDetails extends Component {
       : (otherQuantity === 0 && -1) || price - otherPrice
   }
 
+  renderLoader() {
+    const { displayVertically } = this.props
+
+    const isVertical = displayVertically ? 'VERTICAL' : 'HORIZONTAL'
```
Comment (gugabribeiro) [inline react/ProductDetails.js]: Isn't better to call this entry `orientation`? `isVertical` looks like a boolean content.

**Review by gugabribeiro — CHANGES_REQUESTED** [review]

**Review by gugabribeiro — APPROVED** [review]

**Pull request merged**